Studio/src/Analysis/AnalysisTool.cpp

```diff
@@ -65,6 +66,11 @@ AnalysisTool::AnalysisTool(Preferences& prefs) : preferences_(prefs) {
   connect(ui_->pcaAnimateCheckBox, SIGNAL(stateChanged(int)), this, SLOT(handle_pca_animate_state_changed()));
   connect(&pca_animate_timer_, SIGNAL(timeout()), this, SLOT(handle_pca_timer()));
 
+  //MCA animation  
+  connect(ui_->mcaLevelBetweenButton, SIGNAL(clicked()), this, SLOT(mca_between_radio_toggled()));
```

Comment:
I know there's a lot in the code that haven't been updated, but we should prefer the newer style Qt signals/slots that are compile time checked vs runtime.